.node-version

```diff
@@ -0,0 +1 @@
+v20.11.1
```

Comment:
I think they're using node v18, and I added this in #1472 so we could probably remove this file